compiler/src/dotty/tools/dotc/transform/PCPCheckAndHeal.scala

```diff
@@ -206,6 +206,8 @@ class PCPCheckAndHeal(@constructorOnly ictx: Context) extends TreeMapWithStages(
               tryHeal(prefix.symbol, tp, pos)
             case _ =>
               mapOver(tp)
+        case tp @ TermRef(NoPrefix, _) if !tp.symbol.isStatic && level > levelOf(tp.symbol) =>
```

Comment:
why do we check `level >` here but `level !=` in `healTypeOfTerm`?

Reply:
Thanks for the explanation, I think it'd be helpful to add something like that as a comment in the code

Reply:
Added to the comment.